components/gitpod-cli/cmd/top.go

```diff
@@ -22,7 +23,12 @@ import (
 	"github.com/olekukonko/tablewriter"
 )
 
+var opts struct {
+	Json bool
+}
+
 func outputTable(workspaceResources *supervisor.ResourcesStatusResponse) {
+	fmt.Printf("Workspace Class: %s\n", workspaceResources.WorkspaceClass)
```

Comment:
It can be empty. Actually for now for most users it will be empty.

Reply:
I thought it have default value `default`? not looked into the codebase but try with a new account

Reply:
We need to clarify it. We should show here `Standard`, `Large` or nothin if no workspace class. We should also return human readable representation from info endpoint as well, so all clients display the same, i.e. somethign like:
```
{
  workspaceClass: {
    id: string
    name: string
  }
}

Reply:
@kylos101 @Furisto Could you clarify which values workspace_class can have now and what is human readable name for each please? 🙏

Reply:
supervisor responds with id and ask for details from server in gp-cli? 

https://github.com/gitpod-io/gitpod/blob/9a7119d05dbb721cfd7151a8dd3e68e00b3439ea/components/gitpod-protocol/src/gitpod-service.ts#L317

Reply:
nice 👍

Reply:
Maybe then show something like:
`Standard: Up to 4 vCPU, 8GB memory, 30GB disk`

I'm not sure now whether supervisor should return anything besides workspace class id, since clients can fetch rest as needed from Gitpod API already.

Reply:
for `--json` i would assume it should be `worksapceClass: g1-standard`? cc @Furisto

Reply:
> @kylos101 @Furisto Could you clarify which values workspace_class can have now and what is human readable name for each please? pray

The workspace classes are g1-standard (Standard is the display name), g1-large (Large), default (Standard (old)) and gitpodio-internal-xl (XL (old)). After switching to workspace classes completely default and gitpodio-internal-xl will go away.

> for --json i would assume it should be worksapceClass: g1-standard? cc @Furisto

It should be the display name so 'Standard'. 

Would it help if we changed the WORKSPACE_CLASS environment variable, so that it contains a json document with the information that you need?

Reply:
> Would it help if we changed the WORKSPACE_CLASS environment variable, so that it contains a json document with the information that you need?

I think it would be better that we use getSupportedWorkspaceClasses that UI is consistent across all interfaces including dashboard. Internally in supervisor it is alright to know the technical name.

Reply:
`getSupportedWorkspaceClasses` in golang is not supported yet (not generated), I'll create another PR to add it and ask WebApp team to review